draft-ietf-quic-transport.md

```diff
@@ -693,8 +696,10 @@ continue to use the same Destination Connection ID for 0-RTT packets, see
 
 The version field for protected packets is the current QUIC version.
 
-The packet number field contains a packet number, which increases with each
-packet sent, see {{packet-numbers}} for details.
+The packet number field contains a packet number, which is protected separately
+from the rest of the packet (see {{QUIC-TLS}} for details).  The underlying
```

Comment:
You're using "separately from" here. I would make it the same in other places.

Reply:
Reworded all of these to emphasize *additional* confidentiality protection, rather than *separateness*, which is a tiny bit misleading.  That also neatly avoids the to/from/than problems I seem to have.